provider/job_utils.py

```diff
@@ -70,10 +70,15 @@ def _wait_until_block_job_completed():
 
 
 @fail_on
-def block_job_complete(vm, job_id, timeout=120):
+def job_complete(vm, job_id, timeout=120):
     wait_until_job_status_match(vm, "ready", job_id, timeout)
-    arguments = {"device": job_id}
-    vm.monitor.cmd("block-job-complete", arguments)
+    arguments = {"id": job_id}
+    vm.monitor.cmd("job-complete", arguments)
```

Comment:
Shall we keep the previous usage? not deprecated it at all.
Does the new usage work from a certain qemu version? or the qmp command supports the `block-job-*` and `job-*` in a certain qemu version then we suggest using the new usage `job-*`?

My suggestion is that we could judge whether the current qmp supports the `job-*`, we will use it preferentially if supports it, otherwise using `block-job-*`.

Reply:
Yongxue, since libvirt began to use -blockdev, it had started the usage of 'job-*' instead of 'block-job-*', so I believe we can replace all the qmp commands 'block-job-*' with 'job-*' completely without much concern.

Reply:
yes, refer to the bz, we can replace all

Reply:
@ngu-niny  @zhencliu 
Get it. Thanks for your explanation.

Reply:
job-* start to be used since qemu 3.1, so it's ok for replace all block-job-* (block-job-set-speed not included) by job-* under -blockdev.

Reply:
sorry for the mistake, job-* take effect since qemu 2.12, and block-job-* start to take effect since qemu 1.3.